### DISCUSSION TITLE
addProjectV2ItemById does not work in private repositories

### DISCUSSION THREAD

**Discussion by Bullrich** [community/community#46505] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

[addProjectV2ItemById](https://docs.github.com/en/graphql/reference/mutations#addprojectv2itembyid) is not working on private repositories.

I have been calling the following mutation:
```gql
mutation($project: ID!, $issue: ID!) {
  addProjectV2ItemById(input: {projectId: $project, contentId: $issue}) {
    item {
      id
    }
  }
}
```

and it **does work on public repositories** with public issues, but when I call this same endpoint in a private repository I get the following error:
`GraphqlResponseError: Request failed due to following response errors: - Resource not accessible by integration`.

Is there any other field I need to fill for private repos or is it a bug with GitHub?

I'm using a GitHub app that has the following permissions:
- Repository permissions:
  - [x] Metadata
- Organization permissions
	- Projects
		- [x] Read
		- [x] Write

**Comment by mmtenez** [comment]
[](url) me

**Comment by isla44** [comment]
Hola ustedes tienen una apk llamada salvapantalla de reloj desearia saber el código para abrir su configuración para poder utilizarla mejor

**Reply by CIFTum** [reply]
เราพบปัญหาในการค้นหาคำตอบaddProjectV2ItemById ไม่ทำงานในที่เก็บส่วนตัว #46505 ซึ่งเป็นฟังก์ชันที่ใช้เพื่อเพิ่มรายการใหม่ในโปรเจกต์ V2 ของเรา ปัญหานี้เกิดจากการอัปเดต API ล่าสุดของเรา ที่มีชื่อว่า ProjectV2API และมีเวอร์ชัน 2.1.0 การอัปเดตนี้ได้เปลี่ยนแปลงวิธีการจัดการกับรหัสโปรเจกต์ ที่ต้องใช้พารามิเตอร์ใหม่ เรากำลังทำงานอย่างเต็มที่เพื่อแก้ไขปัญหานี้และจะแจ้งให้ทราบเมื่อมีความคืบหน้า

**Comment by av1934413** [comment]
Thanks
